subscription-service/src/main/kotlin/com/egm/stellio/subscription/service/SubscriptionService.kt

```diff
@@ -591,4 +586,42 @@ class SubscriptionService(
     private var rowToSub: (Row) -> String = { row ->
         row.get("sub", String::class.java)!!
     }
+
+    /**
+     * Function that returns all subscriptions that contain the 'timeInterval' attribute and
+     * when the 'lastNotification' has passed
+     *
+     * @return list of subscription
+     */
+    suspend fun getAllSubscriptions(): List<Subscription> {
```

Comment:
the name of this function is very misleading.

Reply:
maybe `getSubscriptionWithIntervalPast`

Reply:
or `getRecurringSubscriptionsToNotify`?